dashboard/urls.py

```diff
@@ -4,9 +4,11 @@
 from dashboard.views import (
     UserCourseEnrollment,
     UserDashboard,
+    UserPaymentStatus,
 )
 
 urlpatterns = [
     url(r'^api/v0/dashboard/(?P<username>[-\w.]+)/$', UserDashboard.as_view(), name='dashboard_api'),
     url(r'^api/v0/course_enrollments/$', UserCourseEnrollment.as_view(), name='user_course_enrollments'),
+    url(r'^api/v0/program_has_payment/$', UserPaymentStatus.as_view(), name='user_program_payment'),
```

Comment:
Can you change the url from `api/v0/program_has_payment/` to `api/v0/program_payments/`?